frontend/src/common/blocks/ActivityBlock/Note.jsx

```diff
@@ -28,6 +28,9 @@ const Note = ({ object, activity, clickOnContent }) => {
       content = Object.values(content)?.[0];
     }
 
+    //Handle carriage return
+    content = content.replaceAll('\n', '<br>')
```

Comment:
Cette ligne génère une erreur lorsque `content` est null. 
```suggestion
    content = content?.replaceAll('\n', '<br>')
```